.github/workflows/test_with_woodwork_main_branch.yml

```diff
@@ -0,0 +1,61 @@
+name: Test Featuretools with Woodwork main Branch
+on:
+  schedule:
+    - cron: "0 * * * *"
```

Comment:
Won't this be excessive? 
@rwedge Should we lower this or come up with another trigger?

Reply:
@gsheni if we fix the issue with https://github.com/alteryx/gh-action-ci we could have it only actually run if there's been a recent commit